kotsadm/web/src/components/apps/DashboardCard.jsx

```diff
@@ -246,7 +250,8 @@ export default class DashboardCard extends React.Component {
                     isSnapshotInProgress ?
                       <Loader size="16" />
                       :
-                      <span className="status-indicator completed"> Enabled </span>
+                      getingAppLicenseErrMsg ? null :
```

Comment:
we should stop adding `?`, `:` and `null` everywhere. a simple line like this should suffice:
```
!gettingAppLicenseErrMsg && <span className="status-indicator completed"> Enabled </span>
```